### DISCUSSION TITLE
ssh ed25519 pubkey not working

### DISCUSSION THREAD

**Discussion by ByteAndBits** [Security-Onion-Solutions/securityonion#11934] [2.4] [answered]
### Version

2.4.30

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

64

### RAM

512

### Storage for /

15TB

### Storage for /nsm

30TB

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

Hi,

we want to secure ssh with pubkey in ed25519 format. We create the key on windows machine and put the key to the users key file.
But when we connect to SecOnion sshd says on manager:

`debug3: mm_answer_keyallowed: publickey authentication: ED25519 key is not allowed`

We use other linux machines (ubuntu, debian) in our environment and it works out of the box. But SecOnion wiht oracle Linux don´t work.

RSA key works on SecOnion.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by TOoSmOotH** [comment, marked as answer]
Security Onion does not manage the sshd config so you are free to modify to your org's requirements.
